src/ui/sass/modules/_AlternativeVerticals.scss

```diff
@@ -13,7 +13,7 @@ $alternative-verticals-emphasized-font-weight: var(--yxt-font-weight-medium) !de
   background-color: var(--yxt-color-brand-white);
   border: var(--yxt-border-default);
 
-  @media (min-width: var(--yxt-breakpoint-mobile-max)) {
+  @media (min-width: $breakpoint-mobile-max) {
```

Comment:
If someone updates the var(--yxt-breakpoint-mobile-max), those changes won't get reflected here, right? That seems like it could be an issue.

Reply:
With the HH Theme, will someone need to add explicit styling to override a breakpoint? Sounds like there's no way we can support it with CSS vars/Ponyfill.

Reply:
you are right! will remove the breakpoint variables.

with the HH theme, we are dropping the ability to dynamically set media query boundaries due do this css restriction.

they can, however, still override styling inside of media queries by explicitly writing their own